packages/manager/apps/hycu/src/components/Order/PackSelection.tsx

```diff
@@ -1,22 +1,14 @@
-import React, { Dispatch, SetStateAction } from 'react';
-import { useNavigate } from 'react-router-dom';
-import { Trans, useTranslation } from 'react-i18next';
+import React, { Dispatch, ReactNode, SetStateAction } from 'react';
```

Comment:
I'm sorry. I'll have the statement sooner. However, we would like to avoid excessive pooling between the Order page and EditPack.

I've raised that as a confirmation of whether or not it still falls within the quality criteria. We're going to decide tomorrow whether we want to stick to that guideline and make the necessary changes or whether we keep that design for production

I think this component can be smaller to include only the PackSelection field. And keep template, logic and action on the main page